### ISSUE TITLE
vc4: MESA_shading_language_glsl130-like extension

### ISSUE THREAD

**Issue by anholt** [anholt/mesa#22]
X could get a big RGB-antialiased text performance improvement from GL_ARB_blend_func_extended, but we can't expose it without part of GLSL 1.30.  However, we can't support all of 1.30 (instancing, for example).

**Comment by imirkin** [comment]
Note that there's now a GL_EXT_blend_func_extended available in GLES2+, and is already supported by mesa. Not sure what the implications of GLAMOR on GLES2 are.

**Comment by anholt** [comment]
Looks like our tests require GLES3, though.

**Comment by anholt** [comment]
Also, glamor on GLES is super broken, because GLES sucks at readpixels/getteximage.

**Comment by imirkin** [comment]
No, there are _gles2 versions available.

**Comment by imirkin** [comment]
Mesa exposes NV_read_buffer/depth/stencil on GLES - I think Rob added those to enable apitrace to read those out.

**Comment by anholt** [comment]
Yeah, it's lack of format "conversions" that are the problem for glamor.

You're right, there's one exec test built for gles2.
